DigitalLearningSolutions.Web/Helpers/DisplayStringHelper.cs

```diff
@@ -83,5 +83,10 @@ public static string GetTimeStringFromMinutes(int minutes)
         {
             return minutes < 60 ? $"{minutes}m" : $"{minutes / 60}h {minutes % 60}m";
         }
+
+        public static string GetTimeStringForScreenReaderFromMinutes(int minutes)
+        {
+            return minutes < 60 ? $"{minutes} minutes" : $"{minutes / 60} hours {minutes % 60} minutes";

```

Comment:
Just double checking, I'm guessing we are fine with "1 hour 0 minutes" rather than going for just "1 hour" whenever we have a whole number of hours? Same sort of question would apply to the tag content of "1h 0m".

Reply:
I think it's fine where we have data down to the minute. Also the old site displays these as 1h 0m, and it's been to show and tell.